ontology/yaml/resources/HVAC/entity_types/CHWS.yaml

```diff
@@ -133,6 +133,18 @@ CHWS_SWTC_WDPC_CLPM_WFRM:
   - CLPM
   - WFRM
 
+CHWS_SWTC_WDPC_CLPM_WFRM_MWFRM:
```

Comment:
How is makeup water incorporated in the CHWS? I would expect this to be on the CDWS